### DISCUSSION TITLE
Antenna placement

### DISCUSSION THREAD

**Discussion by AlanSamet** [witnessmenow/ESP32-Cheap-Yellow-Display#144] [General]
Just looked at this design (nice work, btw) and am sharing this thing I noticed in the design guide regarding antenna placement that might give you a free performance boost in the future. See section 4.2: https://docs.espressif.com/projects/esp-hardware-design-guidelines/en/latest/esp32/esp-hardware-design-guidelines-en-master-esp32.pdf

**Comment by witnessmenow** [comment]
This isn't something I designed, it's just a cheap off the shelf board available from Aliexpress!
